dev/fattest.simplicity/src/componenttest/rules/repeater/RepeatTests.java

```diff
@@ -67,6 +68,8 @@ public static RepeatTests with(RepeatTestAction action) {
 
     private final List<RepeatTestAction> actions = new ArrayList<>();
 
+    private Optional<Supplier<Boolean>> additionalEvaluationCheck = Optional.empty();
```

Comment:
I don't see a reason why we couldn't have more than one check. Should we have a list here?